src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/Blocks/product_preferences_pagination.html.twig

```diff
@@ -47,6 +47,11 @@
                     {{ form_errors(paginationForm.default_order_way) }}
                     {{ form_widget(paginationForm.default_order_way) }}
                 </div>
+                <div class="form-group">
+                    {{ ps.label_with_help(('Display out-of-stock products last'|trans), ('If enabled, products will be in this order: 1. in-stock, 2. available for order but quantity is 0, 3. out-of-stock. This does not apply when ordering by name & sales'|trans({}, 'Admin.Shopparameters.Help'))) }}
```

Comment:
Hello, can you explain the "ordering by name & sales" please? Also, I would write _If enabled, products will be displayed in this order: 1. in-stock, 2. available for order but quantity is 0, 3. out-of-stock. Note that it does not apply when ordering by name or sales._ instead, what do you think?